test/unit/database_tables.js

```diff
@@ -194,4 +194,288 @@ describe('DatabaseTables', function() {
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tabletwo'},
+                    {
+                        dbname: 'db1',
+                        schema_name: 'public',
+                        table_name: 'analysis_b194a8f896_81cc00c1cfbd5c04d3375fc0e0343a34ae979f34'
+                    }
+                ],
+                skipNotUpdatedAtTables: true,
+                skipAnalysisCachedTables: false,
+                expectedLength: 0
+            },
+            {
+                result: [
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tableone', updated_at: Date.now()},
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tabletwo'},
+                    {
+                        dbname: 'db1',
+                        schema_name: 'public',
+                        table_name: 'analysis_b194a8f896_81cc00c1cfbd5c04d3375fc0e0343a34ae979f34',
+                        updated_at: Date.now()
+                    }
+                ],
+                skipNotUpdatedAtTables: true,
+                skipAnalysisCachedTables: false,
+                expectedLength: 2
+            },
+            {
+                result: [
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tableone', updated_at: Date.now()},
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tabletwo', updated_at: Date.now()},
+                    {
+                        dbname: 'db1',
+                        schema_name: 'public',
+                        table_name: 'analysis_b194a8f896_81cc00c1cfbd5c04d3375fc0e0343a34ae979f34',
+                        updated_at: Date.now()
+                    }
+                ],
+                skipNotUpdatedAtTables: true,
+                skipAnalysisCachedTables: false,
+                expectedLength: 3
+            },
+            {
+                result: [
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tableone', updated_at: Date.now()},
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tabletwo'},
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tablethree', updated_at: Date.now()},
+                    {
+                        dbname: 'db1',
+                        schema_name: 'public',
+                        table_name: 'analysis_b194a8f896_81cc00c1cfbd5c04d3375fc0e0343a34ae979f34',
+                        updated_at: Date.now()
+                    }
+                ],
+                skipNotUpdatedAtTables: true,
+                skipAnalysisCachedTables: false,
+                expectedLength: 3
+            },
+            {
+                result: [
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tableone'},
+                    {dbname: 'db1', schema_name: 'public', table_name: 'tabletwo'},
```

Comment:
I don't think we shall be much concerned about it (as it's been always like that), but that situation happens when a table has been just cartodbfy'ed and there has not been any update on it yet (thus the trigger has not inserted a row in cdb_tablemetadata).

Reply:
After second thought: maybe this is an interesting case to consider. Let's give it some thought before going any further. This may be a bit more tricky than the analysis case IMHO.

Reply:
A question comes to my mind: should `CDB_CartoDBfyTable()` set `now()` for `updated_at` in `CDB_TableMetadata`?

Reply:
That's a good question. Another question in turn: what happens in the caching layers if we issue an invalidation for something that is not cached yet?

Reply:
AFAIK, nothing that we cannot handle. Invalidating a key that doesn't exist in the cache layer doesn't throw any error. Taking a look to the code, we handle errors coming from Cache Provider and we log them.